pylgbst/peripherals.py

```diff
@@ -863,6 +863,56 @@ def _props_msg(self, msg):
             self._notify_subscribers(usbyte(msg.parameters, 0))
 
 
+class RemoteButton(Peripheral):
+
+    # this only works with modes rckey (0), keyr (1), and keya (2) as in
+    # https://virantha.github.io/bricknil/lego_api/lego.html#remote-buttons
+    button_sets = {0: "LEFT",
+                   1: "RIGHT"}
+    button_events = {b'\x01': "PLUS",
+                     b'\x7f': "RED",
+                     b'\xff': "MINUS",
+                     b'\x00': "RELEASE"}
+
+    def __init__(self, parent, port):
+        super().__init__(parent, port)
+
+        self.hub.add_message_handler(MsgHubProperties, self._props_msg)
+
+    def subscribe(self, callback, mode=0x00, granularity=1):
+        # override base class to prevent invalid mode
+        if mode > 2:
```

Comment:
Can you create the mode constants, like it's done in other peripherals? It helps understanding the meaning of available modes.

Reply:
Done